README.md

```diff
@@ -9,7 +9,7 @@ When you purchase a migratory salmon and sea trout rod licence, you are legally
 Users are asked to submit details of the fish they caught (species, weight, number etc) and where they caught them.
 
 ## To build
-Vresion 8.9.1 or above of Node.js is required to run the service 
+Version 14 or above of Node.js is required to run the service, although it works with 8
```

Comment:
This doesn't made sense to me - do you mean you need v14 to develop it, but it will run on v8?

Reply:
I sort of meant to say that version 14 should be used in live and when developing. But it is possible that it will work with 8. However I could just say 'Version 14 or above of Node.js is required to run the service' and leave the v8 part out.

Reply:
Yeah, if you wouldn't mind. Avoids any confusion then.